omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImpl.java

```diff
@@ -49,7 +49,7 @@ public class RestServiceImpl implements RestService {
 	
 	volatile Map<String, ResourceDefinition> resourceDefinitionsByNames;
 	
-	volatile Map<Class<?>, Resource> resourcesBySupportedClasses;
```

Comment:
Did you intentionally change the access modifier?

Reply:
Yes, being used here https://github.com/mherman22/openmrs-module-webservices.rest/blob/08b817fd98e0c15f81f021d4e856b0c2b598c125/omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerGenerationUtil.java#L407

Reply:
It is not a good practice for outsiders to access internal implementation details of a class?

Reply:
True dat. Let me get a finer way of doing it